Table: Add select/unselect all column values to table filter (#79290)
grafana/grafana: docs/sources/panels-visualizations/visualizations/table/index.md

Click the check box next to the values that you want to display. Enter text in the search field at the top to show those values in the display so that you can select them rather than scroll to find them.

> ### Select column values

Reviewer: If I'm understanding correctly, this is a function to select or deselect all the visible values and add them to your filtering. 
```suggestion
### Select all displayed column values
```

Author: Yes, this is the idea, thanks you very much for your changes!

> To select the subset of column values currently displayed.
> 
> Click the **Select values** button to add the current subset of visible values to the full set of filter column values.

Reviewer: I'm trying to simplify the language a bit here. Hopefully I haven't changed the meaning.
```suggestion
Select all displayed column values.

Click the **Select values** button to add all displayed values to the filter.
```

> ### Unselect column values

Reviewer: ```suggestion
### Clear selected column values 
```

> To unselect the subset of column values currently displayed.
> 
> Click the **Unselect values** button to remove the current subset of visible values from the full set of filter column values.

Reviewer: ```suggestion
Remove the all displayed column values from the filter selection.

Click the **Unselect values** button to remove all displayed column values from the filter.
```

Click the check box next to the values that you want to display. Enter text in the search field at the top to show those values in the display so that you can select them rather than scroll to find them.

> Click the check box next to the Ok/Cancel buttons to add or remove all displayed values to/from the filter.

Reviewer: In the screenshot, it looks like the box is positioned above buttons.
```suggestion
Click the check box above the **Ok** and **Cancel** buttons to add or remove all displayed values to/from the filter.
```
